crates/chainspec/src/spec.rs

```diff
@@ -1333,15 +1357,19 @@ Post-merge hard forks (timestamp based):
                 ),
                 (
                     EthereumHardfork::GrayGlacier,
-                    ForkId { hash: ForkHash([0xf0, 0xaf, 0xd0, 0xe3]), next: 1681338455 },
+                    ForkId { hash: ForkHash([0xf0, 0xaf, 0xd0, 0xe3]), next: 15537394 },
+                ),
+                (
+                    EthereumHardfork::Paris,
+                    ForkId { hash: ForkHash([0xbf, 0xe7, 0x0a, 0x5e]), next: 1681338455 },
                 ),
                 (
                     EthereumHardfork::Shanghai,
-                    ForkId { hash: ForkHash([0xdc, 0xe9, 0x6c, 0x2d]), next: 1710338135 },
+                    ForkId { hash: ForkHash([0xa7, 0x01, 0xb4, 0xc9]), next: 1710338135 },
                 ),
                 (
                     EthereumHardfork::Cancun,
-                    ForkId { hash: ForkHash([0x9f, 0x3d, 0x22, 0x54]), next: 0 },
+                    ForkId { hash: ForkHash([0x74, 0x9e, 0x0c, 0xdc]), next: 0 },
```

Comment:
these fork ids should not change. they are used in discovery, so its important they stay fixed to line up with what every other client agrees on